src/SFA.DAS.ApprenticeFeedback.Data/SFA.DAS.ApprenticeFeedback.Data.csproj

```diff
@@ -10,6 +10,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\SFA.DAS.ApprenticeFeedback.Application\SFA.DAS.ApprenticeFeedback.Application.csproj" />
```

Comment:
I don't think this project reference is needed, what was it added for?

Reply:
It was auto added when we were trying things in the investigation as to why the API was not working, will remove that

Reply:
It was auto added when we were trying things in the investigation as to why the API was not working, will remove that